src/pudl/__init__.py

```diff
@@ -23,6 +23,7 @@
 import pudl.extract.excel
 import pudl.extract.ferc1
 import pudl.extract.ferc714
+import pudl.glue.eia_epacems
```

Comment:
@aesharpe Any time you add a new module, if you want it to be available when someone does `import pudl` it needs to end up in here. Otherwise, it has to be specifically imported like `import pudl.glue.eia_epacems`.

Reply:
ahhh thank you, this clears up some confusion I was having!